lib/S3Plugin.js

```diff
@@ -216,21 +243,40 @@ class S3Plugin {
 
     return true;
   }
+  /**
+   * Controller: file
+   * Action: getFilesKeys
+
+   * Get list of files keys from S3 bucket.
+   */
+  async getFilesKeys() {
+    this._assertCredentials();
+    const s3KeyList = await listAllObjects(this.s3, {
+      Bucket: this.config.bucketName
+    });
+    const filesKeys = s3KeyList.map(k => ({
+      ...k,
+      Key: this._getUrl(k.Key)
+    }));
+    return {
+      filesKeys: filesKeys
```

Comment:
Since the key has the same name as the variable, you can use a [shorthand property](https://tylermcginnis.com/shorthand-properties/) (that why I proposed to rename this variable in the first time
```suggestion
      filesKeys
```